ucsmsdk/ucsgenutils.py

```diff
@@ -40,6 +40,15 @@
     "return", "try", "while", "with", "yield"]
 
 
+def str_to_bool(s):
+    """
+    convert case insensitive 'true' string to bool True else False
+    """
+    if s in AFFIRMATIVE_LIST:
+        return True
+    return False
```

Comment:
@ragupta-git the above 3 lines can just be replaced by 1 line.. `return s in AFFIRMATIVE_LIST`